src/python/bot/fuzzers/ml/rnn/utils.py

```diff
@@ -91,7 +91,13 @@ def encode_text(string_data):
   Returns:
     An encoded list of integers representing code points.
   """
-  return list(map(ord, string_data))
+  result = None
+  try:
+    result = list(map(ord, string_data))
```

Comment:
which one is the python2 case, and which one is python 3? maybe we should be explicit and use

```python
if sys.version_info.major == 3:
 etc
```

Reply:
nice, thanks!

Reply:
Another super nit now that I understand what's going on :) 

We should rename "string_data" to "bytes_data", as strings in python 3 are different. In this case we're reading as 'rb' before passing it to this function so we're working on bytes (hence the exception when you try to call `ord`).

Reply:
done!